dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/relationship/hibernate/RelationshipStoreTest.java

```diff
@@ -179,22 +177,22 @@ void testGetByRelationshipType()
     @Test
     void testGetByRelationship()
     {
-        addTeiToTeiRelationShip();
+        Relationship teiRelationship = addTeiToTeiRelationShip();
 
         Optional<Relationship> existing = relationshipService.getRelationshipByRelationship( teiRelationship );
 
         assertTrue( existing.isPresent() );
     }
 
-    void addTeiToTeiRelationShip()
+    private Relationship addTeiToTeiRelationShip()
```

Comment:
I would call this method `relationship()` as you just return a new relationship. You don't really add anything to a relationship

Reply:
this creating/adding a tei -> tei (tei to tei) relationship, i don't see any benefit in renaming that way